pbj-integration-tests/src/main/java/com/hedera/pbj/integration/fuzz/ObjectBuilder.java

```diff
@@ -0,0 +1,188 @@
+package com.hedera.pbj.integration.fuzz;
+
+import com.hedera.pbj.runtime.EnumWithProtoMetadata;
+import com.hedera.pbj.runtime.OneOf;
+import com.hedera.pbj.runtime.io.buffer.Bytes;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.ParameterizedType;
+import java.lang.reflect.Type;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+/**
+ * A factory for Protobuf model objects.
```

Comment:
Is it needed at all? I don't know details, but Jasper mentioned PBJ integration tests already have code to generate all permutations of all HS objects

Reply:
I'd love to reuse that, but I haven't seen the code that would do what I need to do here. If you could point me to that generator, I'd be happy to explore.

Reply:
Thanks for pointing to the ModelTest.ARGUMENTS thing. It fits perfectly here and avoids the complicated Java Reflection code.